src/components/tabBar/TabBarItem.tsx

```diff
@@ -247,12 +270,10 @@ function createStyles() {
     indicator: {
       backgroundColor: INDICATOR_BG_COLOR,
       height: INDICATOR_HEIGHT,
-      marginHorizontal: INDICATOR_SPACINGS
+      marginHorizontal: HORIZONTAL_PADDING
     },
     badge: {
-      marginLeft: 4
+      marginLeft: Spacings.s1
     }
   });
 }
-
-export default asBaseComponent<TabBarItemProps, State>(TabBarItem);
```

Comment:
If it was already using `asBaseComponent`, why would you want to go back and use PureBaseComponent?

Reply:
Done